addons/rating/views/rating_rating_views.xml

```diff
@@ -87,7 +87,7 @@
                                             </span>
                                         </li>
                                         <li>
-                                            on <field name="create_date" />
+                                            on <field name="write_date" />
```

Comment:
What is the purpose of doing it in stable ? It is not a bug ... seems people could do it manually if they want. Especially this is the first time it is reported, updating a view is easy on a production environment.